lasagne/layers/embedding.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import theano.tensor as T
+
+from .. import init
+from .base import Layer
+
+
+__all__ = [
+    "EmbeddingLayer"
+]
+
+
+class EmbeddingLayer(Layer):
+    """
+    A layer for word embeddings. The input should be a `theano.Tensor.ivector``
+    specifying which embeddings to use.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape
+
+    input_size: int
+        The Number of different embeddings. The last embedding will have index
+        input_size - 1.
+
+    output_size : int
+        The size of each embedding.
+
+    W : Theano shared variable, numpy array or callable
+        The embedding matrix.
+
+    Examples
+    --------
+    >>> from lasagne.layers import EmbeddingLayer, InputLayer, helper
+    >>> import theano
+    >>> import theano.tensor as T
+    >>> x = T.ivector()
+    >>> l_in = InputLayer((3, ))
+    >>> W = W=np.arange(3*5).reshape((3, 5)).astype('float32')
+    >>> l1 = EmbeddingLayer(l_in, input_size=3, output_size=5, W=W)
+    >>> output = helper.get_output(l1, x)
+    >>> f = theano.function([x], output)
+    >>> x_test = np.array([0, 2]).astype('int32')
+    >>> print f(x_test)
+    [[  0.   1.   2.   3.   4.]
+     [ 10.  11.  12.  13.  14.]]
```

Comment:
The print statement will fail in Python 3. Can you just do `>>> f(x_test)` instead?